master/internal/kubernetes/pods.go

```diff
@@ -400,6 +400,9 @@ func (p *pods) cleanUpPodHandler(ctx *actor.Context, podHandler *actor.Ref) erro
 		return errors.Errorf("unknown pod handler being deleted %s", podHandler.Address())
 	}
 
+	name := fmt.Sprintf("%s-priorityclass", podInfo.containerID)
+	_ = p.clientSet.SchedulingV1().PriorityClasses().Delete(name, &metaV1.DeleteOptions{})
```

Comment:
Should at least log an error here in case of failure, even if it shouldn't happen in theory.

Reply:
Will add a log statement. 

There is the case where if a user doesn't specify a priority level, we'll automatically user `determined-medium-priority`. If this happens, there is nothing to delete and would return an "error". I can tailor the log statement accordingly.

Reply:
Oh, hmm... if it's not feasible to pass the information of whether or not we used the default option through to this point, I'd just log it at debug level so that the information exists but is ignored in the routine case.